middleware/pprof/pprof_test.go

```diff
@@ -89,7 +89,6 @@ func Test_Pprof_Index_WithPrefix(t *testing.T) {
 }
 
 func Test_Pprof_Subs(t *testing.T) {
-	t.Parallel()
```

Comment:
Why did this get removed?

Reply:
go version go1.20.2 darwin/amd64
mac ventura 13.1
method Test_Pprof_Subs, Test_Pprof_Subs_WithPrefix
`/debug/pprof/profile` run at same time, `cpu profiling already in use` error
![image](https://github.com/gofiber/fiber/assets/7147958/786108a2-0e4a-4aa0-9feb-b3f44e49d3f9)

![image](https://github.com/gofiber/fiber/assets/7147958/d95e2081-bcd2-405c-8781-7afc03f84db3)